odm2api/ODM2/models.py

```diff
@@ -265,6 +265,14 @@ class SamplingFeatures(Base):
     """float: The elevation of the sampling feature in meters, or in the case of Specimen, 
               the elevation from where the SamplingFeature.Specimen was collected"""
     ElevationDatumCV = Column('elevationdatumcv', ForeignKey(CVElevationDatum.Name), index=True)
+    """str: The code for the vertical geodetic datum that specifies the zero point for 
```

Comment:
@lsetiawan are these suppose to be docstrings? Should we consolidate them into a single docstring or make them a comment with a `#`?

The auto-generated docs from this will be quite messy :confused:

Reply:
@ocefpaf I merely followed the google example:

```python
self.attr3 = param3  #: Doc comment *inline* with attribute

#: list of str: Doc comment *before* attribute, with type specified
self.attr4 = ['attr4']

self.attr5 = None
"""str: Docstring *after* attribute, with type specified."""
```

There are three options and I did the last option. What do you think I should do? Thanks.

Reply:
It is not about the example, it is about how we want the docs to look. Some people are on the side that no docs are better than bad automated docs. It is easy to create an automated doc page that is incomprehensible.

I am not saying that is the case here but someone should read the results with care to see if they make sense.